pkg/runner/runner.go

```diff
@@ -77,19 +79,32 @@ func RunBundle(action string, ns string, args []string) {
 		"bundle-action":   action,
 		"bundle-pod-name": pn,
 	}
+
+	// TODO: using edit directly. The bundle code uses clusterConfig.SandboxRole
+	// which is defined by the template. So far we've been using edit.
+
+	runtime.NewRuntime(runtime.Configuration{})
+	targets := []string{ns}
+	serviceAccount, namespace, err := runtime.Provider.CreateSandbox(pn, ns, targets, "edit", labels)
```

Comment:
We should expose ability to use `admin` instead of `edit`. Some APBs I'm testing fail on `edit` sandbox.

Reply:
Do you want that to be a config? What requirements would be needed to allow that? i.e. can any user have admin rights? I guess they can to their own namespaces.